File: java/Human.java
public class Human {
  private String nik;
  private String nama;
  private String jenisKelamin;

  Human(){
    this.nik = "";
    this.nama = "";
    this.jenisKelamin = "";
  }
  Human(String nik, String nama, String jenisKelamin){
    this.nama = nama;
    this.nik = nik;
    this.jenisKelamin = jenisKelamin;
  }

  public String getNama(){
    return this.nama;
  }
  public String getNik(){
    return this.nik;
  }
  public String getJenisKelamin(){
    return this.jenisKelamin;
  }
  public void setNama(String nama){
    this.nama = nama;
  }
  public void setNik(String nik){
    this.nik = nik;
  }
  public void setJenisKelamin(String jenisKelamin){
    this.jenisKelamin = jenisKelamin;
  }
}

File: php/Human.php
<?php
class Human {
  private $nik;// atribut NIK
  private $nama;// atribut Nama
  private $jenisKelamin;// Jenis Kelamin

  public function __construct($nik = "", $nama = "", $jenisKelamin= ""){
    // inisiasi dengan value awal
    $this->nama = $nama;
    $this->nik = $nik;
    $this->jenisKelamin = $jenisKelamin;
  }

  public function getNama(){
    // return nama human
    return $this->nama;
  }
  public function getNik(){
    // return nik human
    return $this->nik;
  }
  public function getJenisKelamin(){
    // rettun jenis kelamin
    return $this->jenisKelamin;
  }
  public function setNama($nama){
    // mengganti value nama
    $this->nama = $nama;
  }
  public function setNik($nik){
    // mengganti value nik
    $this->nik = $nik;
  }
  public function setJenisKelamin($jenisKelamin){
    // mengganti value Jenis Kelamin
    $this->jenisKelamin = $jenisKelamin;
  }
}
